src/main/java/com/rocana/lucene/codec/v1/RocanaSearchCodecV1.java

```diff
@@ -54,4 +92,23 @@ Codec getWrappedCodec() {
     return delegate;
   }
 
+  /**
+   * This codec wraps the {@link Lucene54Codec}, which itself
+   * returns a {@link PerFieldPostingsFormat} that wraps
+   * {@link RocanaLucene50PostingsFormat}.
```

Comment:
this reference is wrong - should point to the Lucene54Codec?  Maybe not, the comment is actually a little hard to follow. Perhaps rephrase?